x11-misc/qps/qps-2.1.0.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake xdg-utils
+
+DESCRIPTION="Qt GUI Process Manager"
+HOMEPAGE="https://lxqt.org/"
+SRC_URI="
+	https://downloads.lxqt.org/downloads/${PN}/${PV}/${P}.tar.xz
+	https://github.com/lxqt/${PN}/releases/download/${PV}/${P}.tar.xz
+"
+
+LICENSE="GPL-2 GPL-2+ LGPL-2.1+ QPL-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="debug"
```

Comment:
This doesn't seem to be doing anything?

... unless it somehow interacts with *cmake.eclass* that I'm not aware of?

Reply:
This previously made cmake.eclass add -DNDEBUG flag but it was dropped in commit 95577dd5076a8e9864e82879fd3af97cf63fcfe9

Reply:
Will look into this one as well.

Reply:
Seems like the only reference I find, '#ifndef NDEBUG', in the sources is commented out, so I guess I'll drop the 'IUSE' from this one.